src/utils/cdk.ts

```diff
@@ -77,6 +77,8 @@ export class CdkBuilder {
   }
 
   addParams(): void {
+    if (!Object.keys(this.template.Parameters).length) return;
```

Comment:
One-liner if-statements 😲

Reply:
Not a fan? I quite like it in select cases, like this one, where you just want to do `return` or `continue` at the start of something. Not a strongly held opinion though so happy to change.